pkg/setting/setting.go

```diff
@@ -143,6 +143,7 @@ var (
 	ExternalUserMngInfo     string
 	OAuthAutoLogin          bool
 	ViewersCanEdit          bool
+	HiddenUsers             map[string]struct{}
```

Comment:
Add to Cfg struct instead, we do not want more global variables

Reply:
I updated it! It's a bit hard to access settings in the database layer as functions are not part of the SqlStore service.